ultimatethumb/utils.py

```diff
@@ -27,10 +32,9 @@ def get_cache_key(key):
 
 def get_thumb_name(source, **options):
     source_name, source_ext = os.path.splitext(os.path.basename(source))
-    data = SortedDict()
+    data = OrderedDict()
     data['source'] = source
-    data['opts'] = SortedDict(options)
-    data['opts'].keyOrder = sorted(data['opts'].keyOrder)
+    data['opts'] = OrderedDict(sorted(options.items()))
```

Comment:
I think the should have an explicit sort key to make sure only the dict key is used for sorting. key=operator.itemgetter(0) or key=lambda i: i[0]. Its your choice ;-)